python/eos/analysis_file.py

```diff
@@ -441,6 +454,43 @@ def validate(self):
             print(m)
         return messages
 
+    def find_configuration(self, task_name:str, task_args:dict):
+        """Find configurations (if any) that match the task and arguments provided.
```

Comment:
Specify the return values in the docstring

Reply:
Resolved.